source/_components/device_tracker.bt_smarthub.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "BT Smart Hub"
+description: "Instructions on how to integrate BT Smart Hub router into Home Assistant."
+date: 2018-10-05 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
+ha_category: Presence Detection
+---
+
+
+This platform offers presence detection by looking at connected devices to a [BT Smart Hub](https://en.wikipedia.org/wiki/BT_Smart_Hub) based router.
+This router is sometimes referred to as the BT Home Hub 6.
+
+To use a BT Smart Hub router in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: bt_smarthub
+    host: 192.168.1.254
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.